src/CrispFieldsModalBase.js

```diff
@@ -144,12 +145,10 @@ class CrispFieldsModalBase extends React.Component {
                 }
                 showTimeSelect={column.type === 'Time'}
                 timeFormat='HH:mm:ss'
-                dateFormat={`dd/MM/yyyy${
-                  column.type === 'Time' ? ', HH:mm:ss' : ''
-                }`}
-                placeholderText={`Select${
-                  rangeEnabled ? ' start ' : ' '
-                }${placeholder}`}
+                dateFormat={`MM/dd/yyyy${column.type === 'Time' ? ', HH:mm:ss' : ''
```

Comment:
Are you able to pull this date format from the browser's localization? This works for the US, but we have a client in Switzerland that this would need to be `dd/MM/yyyy`